tools/build_defs/framework.bzl

```diff
@@ -378,7 +384,10 @@ def cc_external_rule_impl(ctx, attrs):
     ]
     output_groups[attrs.configure_name + "_logs"] = wrapped_files
     return [
-        DefaultInfo(files = depset(direct = rule_outputs + wrapped_files)),
+        DefaultInfo(
+            files = depset(direct = rule_outputs + wrapped_files),
+            runfiles = ctx.runfiles(ctx.files.data),
```

Comment:
I think you also need to grab the transitive run files from ctx.files.data and ctx.files.deps as per the example at https://docs.bazel.build/versions/master/skylark/rules.html#runfiles?

Reply:
Good call, updated.

Reply:
I've also added the default runfiles of `ctx.attr.lib_source` to the set being returned in the `DefaultInfo` provider here. I think that's correct as well(?)

Reply:
Yes I think that is correct; do we also need `ctx.attr.additional_inputs` run files too?

Reply:
Another good call, updated.